packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -245,7 +245,7 @@ export default compose( [
 			hasDescendants: !! getClientIdsOfDescendants( [ clientId ] ).length,
 		};
 	} ),
-	withDispatch( ( dispatch, ownProps ) => {
+	withDispatch( ( dispatch, ownProps, registry ) => {
```

Comment:
I think the only suggestion I'd have would be to maybe consider destructuring `registry` to just pick `select`?

Reply:
I had some advice from @mcsf that Gutenberg prefers being explicit about where you are accessing the properties of data controls. I took this as avoiding unnecessary destructuring in/around `select`/`dispatch`/`registry`. I could be wrong.

Reply:
> I had some advice from @mcsf that Gutenberg prefers being explicit about where you are accessing the properties of data controls. I took this as avoiding unnecessary destructuring in/around `select`/`dispatch`/`registry`. I could be wrong.

Thanks for thinking about my advice 😄. In this case I don't really have an opinion, since, in the context of `withDispatch`, `select` could only come from one place.

My main point — which was merely stylistic — in the other PR was about not extracting the selectors into their own variable, e.g. `const { getFoos } = select( store )`, and instead always calling `select( store ).getFoos( ...args )`.